### PR TITLE
Multi node

### PR DESCRIPTION
**Pull request by rongzha1**
test single/multi-node use unified files && cmd (using different parameters n)
env:
source ../tools/teamforge/ml-dlmsl/_install/intel64/bin/mlslvars.sh
source ../tools/impi_2017_3/bin64/mpivars.sh
unset I_MPI_PMI_LIBRARY

run cmd
single:
MLSL_NUM_SERVERS=0 MLSL_HOSTNAME_TYPE=1 I_MPI_FABRICS=tmi mpirun -n 1 -ppn 1 -hosts mlt-bdw017 python googlenet_benchmark.py
mul:
MLSL_NUM_SERVERS=0 MLSL_HOSTNAME_TYPE=1 I_MPI_FABRICS=tmi mpirun -n 2 -ppn 1 -hosts mlt-bdw017,bdw018 python googlenet_benchmark.py

### CODE REVIEW

**Comment by pengzhao-intel** [comment]
How about moving the code into the new directory, like googlenet_v1 since it can work for either single and multiple modes?

Does "python googlenet_benchmark.py" work?

**Comment by rongzha1** [comment]
OK,  what about diretory name: googlenet_vN ?   for N=1~n
python googlenet_benchmark.py  can work well.
But single node also depend on msls lib. Now ,we use msls lib to get (rank,size), if size >1, use multi code.

Maybe we can remove mlsl lib dependency of single node  in phase 2 .

File: democase/googlenet_v1_mn/googlenet_benchmark.py [hunk 1]
```diff
@@ -1,66 +1,81 @@
...
+    (train_model, shared_x, shared_y, shared_lr) = compile_train_model(model, batch_size = batch_size,multinode_mode = multinode_mode)
+    (validate_model, shared_x, shared_y) = compile_val_model(model, batch_size = batch_size)
+
+    images = np.random.random_integers(0, 255, input_shape).astype('float32')
+    labels = np.random.random_integers(0, 999, batch_size).astype('int32')
+    shared_x.set_value(images)
+    shared_y.set_value(labels)
+    #set_learning_rate(shared_lr, iter)
+
+    if multinode_mode == False:
```
Comment (ciyongch) [inline democase/googlenet_v1_mn/googlenet_benchmark.py]: Should it be "drop_flag = False" ?

**Comment by ciyongch** [comment]
Remove all the useless debug codes (like print...). 
googlenet_v1 stands for v1 topology not the number of nodes.

**Pull request merged**